rgb-format.md

```diff
@@ -0,0 +1,101 @@
+# RGB format description
+
+## RGB colour properties
+
+Complete set of properties relating to an RGB colour:  
+- Components
+- Encoding
+- Block compression/chroma subsampling strategy
+- Gamma compression strategy
+- Colour space
+- Dynamic range
+- Other encoding options
+    - Swizzling, Endian, etc
+
+## RGB format string encoding:
+
+`"components_[format]_[colourspace]_["BE" big-endian]_[swizzle parameters]_[userdata]"`
+
+### `components`:  
+A set of single character component names:
+- `r` `g` `b` - red, green, blue
+- `l` - luma/luminance (`r = g = b`)
+- `a` - alpha (exempt from colourspace)
+- `e` - shared exponent
+- `x` - unused bits
+
+Component names are lower-case. (or should they be upper case? they could be case-insensitive, but then we halve the namespace for future expansion)
+
+### `format` [optional]:  
+Description of the format for each channel, underscore separated for each component
+Eg: `rgb_8_8_8`, `bgra_10_10_10_2`, `rgbe_9_9_9_5`, `rgba_s6_f11_4.8_u3`  
+Encoding options:
+- Omitted: 8 bit normalised integer for each compoent
```

Comment:
typo in `component`

Reply:
Thanks!

Reply:
https://github.com/DlangGraphicsWG/documents/pull/16